src/configuration.ts

```diff
@@ -37,9 +37,12 @@ export class ConnectionConfiguration implements IConnectionConfiguration {
         if (process.env['PROCESSOR_ARCHITEW6432'] === 'AMD64') {
           programFiles = process.env['ProgramW6432'] || 'C:\\Program Files';
         }
+        // On Windows we have a subfolder called 'Puppet' that has
+        // every product underneath
         return path.join(programFiles, 'Puppet Labs', 'Puppet');
       default:
-        return '/opt/puppetlabs/puppet';
+        // On *nix we don't have a sub folder called 'Puppet'
+        return '/opt/puppetlabs';
```

Comment:
This was `/opt/puppetlabs/puppet` before, which would make `puppetDir` `/opt/puppetlabs/puppet/puppet` which doesn't exist